cfgov/v1/atomic_elements/atoms.py

```diff
@@ -99,9 +107,13 @@ class ImageBasic(blocks.StructBlock):
     alt = blocks.CharBlock(required=False)
 
     def __init__(self, required=True):
-        self.required = required
+        self.is_required = required
         super(ImageBasic, self).__init__()
 
+    @property
+    def required(self):
+        return self.is_required
+
```

Comment:
What's the point of continuing the `required` property? Instead of replacing its use with `is_required`.